client/ayon_core/hosts/maya/plugins/publish/validate_vrayproxy.py

```diff
@@ -17,18 +20,18 @@ def process(self, instance):
         if not self.is_active(data):
             return
         if not data["setMembers"]:
-            raise KnownPublishError(
+            raise PublishValidationError(
```

Comment:
Should all these be PublishValidationError? User can't do anything about it...

Reply:
I wasn't entirely sure - it felt like something the artists did have control over. 

- If there's no set members, then they just added no objects to the instance and they should add it.
- If frame end is below frame start they should fix it on the instance
- If both vrmesh and alembic are disabled for extraction - that'd also be a toggle for the artist to fix.

Not saying that the validation message is perfect the way they are, but at least I feel like these are something _for the artist_ that they can handle?

Reply:
I don't know the logic behind, but the message should be modified, at least remove the "This is a bug". If it is a bug it is not artists issue...

Reply:
I agree that this is something artist has power to change, but I would remove the "bug" sentence also.